src/components/Summary.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import _ from 'lodash';
+
+import SummaryScope from './SummaryScope.js';
+
+const Summary = React.createClass({
+  getInitialState() {
+    return {summary: []};
+  },
+  componentDidMount() {
+    this.fetchScore.call(this);
+    setInterval(() => {
+      this.fetchScore.call(this);
```

Comment:
We should think about an implementation with `setTimeout` instead, but for now it works and is fine. Esp since we might swap this out for web sockets.